src/components/presentational/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,41 @@
+import "./ProgressBar.css";
+import React, { ReactElement, useEffect } from "react";
+
+export interface ProgressBarProps {
+	background?: string;
```

Comment:
We use color suffix for color properties, so would use backgroundColor and fillColor.

Also would make both background/fill a ColorDefinition and not a string - that was probably introduced after this PR but see the button component for an example (ColorDefinition allows specifying 2 colors, one for light mode and one for dark mode)

Reply:
Oh hmm, i'm realizing that fill might actually be a gradient...yet let me think about that for a sec